packages/apify/src/enqueue_links/click_elements.ts

```diff
@@ -253,6 +257,7 @@ function createTargetCreatedHandler(page, requests) {
         // possible errors like target closed.
         try {
             const createdPage = await target.page();
+            // @ts-ignore TODO: Object is possibly 'null'. Using @ts-expect-error fails the test
```

Comment:
Btw, if we know the createdPage prop is always present, you can add a `!` in front of the `.` to tell TS that it cannot be null

Reply:
I am not sure, but I think it could actually be `null`, looking at the comment above:
// We want to close the page but don't care about
// possible errors like target closed.
I gues for some internal browser/network issues, and that's exactly the reason why it's wrapper into try/catch, but I might be wrong.